tools/please_go_test/gotest/write_test_main.go

```diff
@@ -236,9 +244,7 @@ func main() {
         args = append(args, "-test.run", testVar)
     }
     os.Args = append(args, os.Args[1:]...)
-	benchmarks := []testing.InternalBenchmark{}
-	var examples = []testing.InternalExample{}
-	m := testing.MainStart(testDeps, tests, benchmarks, examples)
+	m := testing.MainStart(testDeps, tests, nil, examples)
```

Comment:
Detecting & running benchmark functions isn't hard in this framework - we just don't have a good way of handling it since they're clearly not a yes/no test.

Maybe we need `plz bench`?